curryer/compute/spatial.py

```diff
@@ -20,12 +20,14 @@
 
 import logging
 import time
+import warnings
 from typing import Union
 
 import numpy as np
 import pandas as pd
 import xarray as xr
 from spiceypy.utils.exceptions import NotFoundError, SpiceyError
+from typing_extensions import deprecated
```

Comment:
This is an external library [typing-extensions](https://pypi.org/project/typing-extensions/) that allows for newer typing features to work in python 310. We would need to add this to the pyproject.toml right? Weirdly I see this library show up in some poetry.lock files locally... but I think we'd need to add it to the toml for consistency.